db/migrate/20190131170022_create_reports.rb

```diff
@@ -0,0 +1,16 @@
+class CreateReports < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reports, id: :uuid, default: -> { "uuid_generate_v4()" } do |t|
+      t.uuid :partner_id
+      t.references :uploaded_by, index: true, foreign_key: { to_table: :publishers }, type: :uuid
+
+      t.string :amount_bat
```

Comment:
IMO amounts being discussed and paid out should be a separate table. We can also track the state in the separate model too. 

Would you rename this column to `partner_requested_amount_in_bat`? Additionally, create another column called `admin_paid_out_in_bat`.

Reply:
Actually now that I think about it, this info should be stored in `invoices`. A `Report` will always have an `Invoice`, but the `Invoice` could or could not contain the `partner_requested_amount_in_bat`. The `revised` state of the `Invoice` will contain the admin amount.